lv_conf_template.h

```diff
@@ -187,6 +187,9 @@
  * but does not guarantee the same rendering quality as the software. */
 #define LV_VG_LITE_USE_BOX_SHADOW 0
 
+/* VG-Lite gradient image maximum cache number. */
+#define LV_VG_LITE_GRAD_CACHE_SIZE 32
```

Comment:
Is it possible to estimate the memory usage? Or at least give a hint.

Reply:
Done. The memory usage of a single gradient image is 4K bytes.